internal/serverinstall/nomad.go

```diff
@@ -285,17 +299,21 @@ func (i *NomadInstaller) Install(
 	s.Update("Waypoint server ready")
 	s.Done()
 
-	if i.config.consulService {
+	switch i.config.serviceProvider {
+	case "consul":
 		s = sg.Add("The CLI has been configured to automatically install a Consul service for\n" +
 			"the Waypoint service backend and ui service in Nomad.")
 		s.Done()
-	} else {
-		s = sg.Add(
-			" Waypoint server running on Nomad is being accessed via its allocation IP and port.\n" +
-				"This could change in the future if Nomad creates a new allocation for the Waypoint server,\n" +
-				"which would break all existing Waypoint contexts.\n\n" +
-				"It is recommended to use Consul for determining Waypoint servers IP running on Nomad rather than\n" +
-				"relying on the static IP that is initially set up for this allocation.")
+	case "nomad":
+		s = sg.Add("The CLI has been configured to automatically install a Nomad service for\n" +
+			"the Waypoint service backend and ui service in Nomad.")
+		s.Done()
+	default:
+		s = sg.Add("Waypoint server running on Nomad is being accessed via its allocation IP and port.\n" +
+			"This could change in the future if Nomad creates a new allocation for the Waypoint server,\n" +
+			"which would break all existing Waypoint contexts.\n\n" +
+			"It is recommended to use Consul for determining Waypoint servers IP running on Nomad rather than\n" +
+			"relying on the static IP that is initially set up for this allocation.")
```

Comment:
Hmm, these three outputs might be better suited for `ui.Output` instead of a step group since they are single output! I think @evanphx has mentioned to be before that doing `\n` new lines in step groups might be a bad idea too.

Reply:
We were doing it previously, so I think it's fine. If it's weird we should probably just fix it so it works right too.